west: update revision for mcumgr library

de-nordic opened the pull request:
The snapshot update adds support for flash devices that have 0
as the erase value.

Signed-off-by: Dominik Ermel <dominik.ermel@nordicsemi.no>

de-nordic committed:
west: update revision for mcumgr library

The snapshot update adds support, to mcumgr, for flash devices that
have non-0xff erase value.

Signed-off-by: Dominik Ermel <dominik.ermel@nordicsemi.no>

nvlsianpu reviewed (commented):
@de-nordic Can the release-note be updated about mcumgr changes?

de-nordic commented:
> 
> 
> @de-nordic Can the release-note be updated about mcumgr changes?

Yes.

de-nordic committed:
doc/releases: Add not on mcumgr support for non-0xff erase flash

The commit adds information, to release notes, on mcumgr now supporting
flash devices that have non-0xff erase value.

Signed-off-by: Dominik Ermel <dominik.ermel@nordicsemi.no>

de-nordic commented:
@nvlsianpu I have added the release note as requested. I admit this is good idea to do this with changes. Thanks!
